keys/employees-keys/README.md

```diff
@@ -116,10 +133,159 @@ sub   4096R/CC8BE1D5 2021-04-02
 
 The key ID in this case is `7429663E`.
 
-## How to export your key
+### Sending key for signing
+
+Generate ASCII armor public key file:
 
 ```
 gpg --armor --output your-name-key.asc --export "YOUR KEY ID HERE"
 ```
 
-Share `your-name-key.asc` textual file as the public portion of your key.
+Obtain fingerprint of your key:
+
+```
+gpg -n -q --import --import-options import-show your-name-key.asc
+```
+
+Armored public key should be sent to
+[3mdeb chat web-of-trust channel](https://chat.3mdeb.com/team-3mdeb/channels/web-of-trust).
+
+Please send fingerprint to 3mdeb Team Leaders or Management using different
+channel then 3mdeb chat e.g. https://keybase.io, LinkedIn, email, etc.
+Fingerprints should be sent to the person who works with you on signing your keys.
+
+### Identity confirmation
+
+Following information is for 3mdeb Team Leaders or Management.
+
+* If official contract between employee and 3mdeb was signed, then national id
+  verification was involved. In that case Team Leader or Manager, based on
+  previous video call or live discussion during recruitment process, can confirm
+  that employee sending keys is the same person who were hired. Also new employee
+  should be the only one who has access to `@3mdeb.com` emails address based on
+  secure credential passing procedure performed by IT.
+* If we establish keys for freelancer, who's identity cannot be confirmed by
+  national id, video call or face to face discussion, we rely on multiple
+  Internet identities e.g. reddit, mailing list, Slack, Wire, Google email,
+  keybase, cryptocurrency address etc. Freelancer have to confirm at least
+  through 5 channels that he/she is the only person in control of channels.
+
+We recommend to create [Keybase account](https://keybase.io) to simplify
```

Comment:
Keybase is owned by Zoom and isn't completely open-source - https://github.com/keybase/client/issues/6374 , not sure if we could trust them TBH

Reply:
@mikebdp2 I would never treat them as a single verification entity, this is just one that may add to social proof identity confirmation. What would you suggest? keyoxide or maybe there is really something nice we can selfhost?

Reply:
I added one sentence to improve that section.

Reply:
@pietrushnic yes, Keyoxide seems nice on paper and its source code is open. Another alternative could be an IndieWeb profile [as described here](https://figbert.com/posts/how-to-replace-keybase-in-three-easy-steps/), although seems too much hassle if you don't have IndieWeb already. Guess it's up to a user to decide the set of services to conveniently & securely proof their identity, and up to you to see if this joint proof is trustworthy or more is needed.